Testing/Data/SystemTest/PEARL/Calibration_Test/Calibration/pearl_absorp_sphere_10mm_newinst2_long.nxs.md5

```diff
@@ -0,0 +1 @@
+d1b66c634fc4e6046e8f75121b505518
```

Comment:
Yes, that sounds good and it will be much clearer what files are used in which test than if all the files go to the same directory.